docs/common/install_and_configure_nginx_with_rtmp_module_on_windows.md

```diff
@@ -60,12 +179,221 @@ stream {
         proxy_pass live;
         ssl_certificate c:\\certs\\fullchain.pem; #root path where your certificate is located e.g.: C:\certs\fullchain.pem
         ssl_certificate_key c:\\certs\\privkey.pem; #root path where your certificate key is located e.g.: C:\certs\privkey.pem
-
         allow all;              # this is public (this is also the default)
     }
 }
 
 rtmp {
+    server { 
+        listen 127.0.0.1:2940;
```

Comment:
**question**: I noticed that for RTMPS we are adding the localhost IP in `listen`, but for RMTP we are just using the port number. Is there a reason behind this?

Reply:
RTMPS is adding the localhost IP because it is consumed by the stream server that listens for all IPs and enables SSL. In RTMP the streaming server is not needed because SSL is not enabled.